packages/common/test/HdWallet.test.ts

```diff
@@ -141,3 +171,21 @@ test('Testnet attribute commitment randomness', () => {
         )
     );
 });
+
+test('Testnet CredId matches credDeployment test', () => {
+    const wallet = ConcordiumHdWallet.fromHex(
+        'efa5e27326f8fa0902e647b52449bf335b7b605adc387015ec903f41d95080eb71361cbc7fb78721dcd4f3926a337340aa1406df83332c44c1cdcfe100603860',
+        'Testnet'
+    );
+    expect(
+        wallet.getCredentialId(0, 0, 1, {
+            onChainCommitmentKey:
+                'b14cbfe44a02c6b1f78711176d5f437295367aa4f2a8c2551ee10d25a03adc69d61a332a058971919dad7312e1fc94c5a8d45e64b6f917c540eee16c970c3d4b7f3caf48a7746284878e2ace21c82ea44bf84609834625be1f309988ac523fac',
+        })
+    ).toEqual(
```

Comment:
From the name of the test I would expect this to verify against the function that generates the credential deployment stuff, and not a hardcoded value.

Reply:
I had just copied the value from the credential deployment test, but I've changed it to calculate it.